js/utils.ts

```diff
@@ -0,0 +1,33 @@
+/** getting random number from stated range
+ *
+ * @param a :minimal of range
+ * @param b :maximum of range
+ * @returns random number from the stated range
+ */
+const getRandomNumber = (a:number, b:number) => {
+	const lower = Math.ceil(Math.min(a, b));
+	const upper = Math.floor(Math.max(a, b));
+	const result = Math.random() * (upper - lower + 1) + lower;
+	return Math.floor(result);
+};
+
+/**getting random element from stated array
+ *
+ * @param elements - array from which random ellement should be taken
+ * @returns return random array ellement
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const getRandomArrayElement = (elements:any[]) => elements[getRandomNumber(0, elements.length - 1)];
```

Comment:
Тут сложная тема Generic'ов, но правильная типизация такая
```suggestion
const getRandomArrayElement = <El>(elements: El[] | readonly El[]) => elements[getRandomNumber(0, elements.length - 1)];
```